src/ui/components/HostPermissions/index.js

```diff
@@ -108,20 +108,6 @@ export class HostPermissionsBase extends React.Component<InternalProps> {
         />,
       );
     }
-    if (permissions.length > 4) {
-      // Replace the final individual permission with a "too many" permission.
-      hostPermissions[3] = (
-        <Permission
-          type="hostPermission"
-          description={this.getPermissionString({
-            messageType,
-            param: permissions.length - 3,
-            multiple: true,
```

Comment:
So do we actually need this `multiple` logic in `getPermissionString()`?

Reply:
We do not need it any more.